.drone.yml

```diff
@@ -98,7 +98,7 @@ name: AppImage
 
 steps:
 - name: build
-  image: nextcloudci/client-5.12:client-5.12-9
+  image: ghcr.io/nextcloud/continuous-integration-client-appimage:client-appimage-1
```

Comment:
I think it would be better to migrate all images to github

that would solve the build failure due to docker rate limits and would be easier to maintain and understand

Reply:
Is this something we want to do in this pr? I would prefer to postpone it to a second pr.

Reply:
I guess that I would be afraid that we forget to do it
this is the reason why I am asking
you do not want to do it now because it is not yet published on github ?